src/main/java/com/mraof/minestuck/block/plant/SpecialCactusBlock.java

```diff
@@ -33,6 +34,6 @@ public boolean canSurvive(BlockState state, LevelReader level, BlockPos pos)
 	
 	protected boolean isSustainableSoil(BlockState soil)
 	{
-		return soil.is(this) || soil.is(BlockTags.SAND);
+		return soil.is(this) || soil.is(MSTags.Blocks.WOOD_TERRAIN_BLOCKS) || soil.is(BlockTags.PLANKS);
```

Comment:
Two things of note:
- The class name still signifies that the class can be used for other cacti blocks, but this change seems to be made specifically with the wooden cactus in mind. Something should be changed.
- The wooden cactus is still being used and placed on sand for the cactus pyramid center in sand land villages. Something should be changed.

Reply:
wrt wooden cactus in sand lands, is that a specifically designed thing or does it call on whatever's under SpecialCactusBlock?

Reply:
I'm not sure what you're asking.
A sand land structure is specifically placing some wooden cacti on the land surface block, which for such a land is sand.

Reply:
SpecialCactusBlock is only used for Wooden Cactus. Change anyway or keep as is?

Reply:
My quick suggestion for that is to just rename the class.